fog/ledger/enclave/src/lib.rs

```diff
@@ -187,6 +190,81 @@ impl LedgerEnclave for LedgerSgxEnclave {
         let outbuf = self.enclave_call(&inbuf)?;
         mc_util_serial::deserialize(&outbuf[..])?
     }
+
+    fn ledger_store_init(&self, ledger_store_id: ResponderId) -> Result<NonceAuthRequest> {
```

Comment:
:thought_balloon: I don't think it should be done in this PR, and I need to better understand the 2 `Deserialize` traits, https://serde.rs/lifetimes.html#trait-bounds
But something about the `inbuf` and `outbuf` repeated everywhere seemed to gnaw at me
I wonder if something like this provides any benefit?
```rust
impl LedgerSgxEnclave {
    fn call_wrapper<T>(&self, call: &EnclaveCall) -> Result<T>
        where
            T: DeserializeOwned,
    {
        let inbuf = mc_util_serial::serialize(call)?;
        let outbuf = self.enclave_call(&inbuf)?;
        mc_util_serial::deserialize(&outbuf[..])?
    }
}

/// Proxy API for talking to the corresponding implementation inside the
/// enclave.
impl LedgerEnclave for LedgerSgxEnclave {
    fn enclave_init(&self, self_id: &ResponderId, desired_capacity: u64) -> Result<()> {
        self.call_wrapper(&EnclaveCall::EnclaveInit(
            self_id.clone(),
            desired_capacity,
        ))?
    }
```